src/NaturalLanguage/NaturalLanguageClient.php

```diff
@@ -0,0 +1,337 @@
+        }
+
+        $this->connection = new Rest($this->configureAuthentication($config));
+    }
+
+    /**
+     * Finds named entities (currently finds proper names) in the text, entity
+     * types, salience, mentions for each entity, and other properties in the
+     * document.
+     *
+     * Example:
+     * ```
+     * $annotation = $language->analyzeEntities('Google Cloud Platform is a powerful tool.');
+     *
+     * foreach ($annotation->entities() as $entity) {
+     *     echo $entity['type'];
+     * }
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeEntities Analyze Entities API documentation
+     * @codingStandardsIgnoreEnd
+     *
+     * @param string|Object $content The content to analyze.
+     * @param array $options {
+     *     Configuration options.
+     *
+     *     @type string $type The document type. Acceptable values are
+     *           `PLAIN_TEXT` or `HTML`. Defaults to `PLAIN_TEXT`.
+     *     @type string $language The language of the document. Both ISO
+     *           (e.g., en, es) and BCP-47 (e.g., en-US, es-ES) language codes
+     *           are accepted. Defaults to English.
+     *     @type string $encodingType The text encoding type used by the API to
+     *           calculate offsets. Acceptable values are `NONE`, `UTF8`,
+     *           `UTF16` and `UTF32`. Defaults to `UTF8`.
+     * }
+     * @return Annotation
+     */
+    public function analyzeEntities($content, array $options = [])
+    {
+        return new Annotation(
+            $this->connection->analyzeEntities(
+                $this->formatRequest($content, $options)
+            )
+        );
+    }
+
+    /**
+     * Analyzes the sentiment of the provided document. Currently only supports
+     * English text.
+     *
+     * Example:
+     * ```
+     * $annotation = $language->analyzeSentiment('Google Cloud Platform is a powerful tool.');
+     * $sentiment = $annotation->sentiment();
+     *
+     * if ($sentiment['polarity'] > 0) {
+     *     echo 'This is a positive message.';
+     * }
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeSentiment Analyze Sentiment API documentation
+     * @codingStandardsIgnoreEnd
+     *
+     * @param string|Object $content The content to analyze.
+     * @param array $options {
+     *     Configuration options.
+     *
+     *     @type string $type The document type. Acceptable values are
+     *           `PLAIN_TEXT` or `HTML`. Defaults to `PLAIN_TEXT`.
+     *     @type string $language The language of the document. Both ISO
+     *           (e.g., en, es) and BCP-47 (e.g., en-US, es-ES) language codes
+     *           are accepted. Defaults to English.
+     * }
+     * @return Annotation
+     */
+    public function analyzeSentiment($content, array $options = [])
```

Comment:
Let's circle back on this once we get a better idea of how people are using the API.